sphinx/source/docs/first_steps/installation.rst

```diff
@@ -37,6 +37,14 @@ method. Otherwise, use ``pip``.
 
         conda install bokeh
 
+    Alternatively, if you want to make sure you always have the most recent
+    version of Bokeh, install from the
```

Comment:
do we want to try to explain here that the version from defaults sometimes takes longer to update to latest? I.e. an indication of why you might choose the bokeh channel?

Reply:
I'd say that "if you want to make sure you always have the most recent version of Bokeh" does make clear why one would want to chose the Bokeh channel. I'm trying to keep things short here, but if this wording is unclear, I could add some more details!

Reply:
Well, I guess the thing (and maybe is not worth splitting hairs over) is that (at least to me) defaults and/or conda-forge are always preferable to use unless there is a reason not to, and basically the only reason is if you need to install with conda right after a release

Reply:
I added a qualifyer about new releases, but for the sake of keeping things as sort as possible, I don't think we need to add much more detail here.